src/libmongoc/tests/test-mongoc-change-stream.c

```diff
@@ -2774,6 +2774,6 @@ test_change_stream_install (TestSuite *suite)
    TestSuite_AddMockServerTest (
       suite, "/change_streams/prose_test_18", prose_test_18);
 
-   test_framework_resolve_path (JSON_DIR "/change_streams", resolved);
+   test_framework_resolve_path (JSON_DIR "/change_streams/legacy", resolved);
```

Comment:
In resyncing the change_streams tests, there are now two subdirectories (`unified` and `legacy`). The `unified` tests are installed in the unified test runner: `runner.c`.